db/migrate/20170703173452_add_miq_expression_enable_and_text_disable.rb

```diff
@@ -0,0 +1,6 @@
+class AddMiqExpressionEnableAndTextDisable < ActiveRecord::Migration[5.0]
+  def change
+    add_column :custom_buttons, :miq_expression_enable, :text
```

Comment:
We've been quite inconsistent with naming (thinking `condition`, `filter`, `expression`), but I think `enablement_<one of those three>` sounds good to me